app/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# 
+FROM python:3.9
+
+# 
+WORKDIR /code
+
+# 
+COPY ./requirements.txt /code/requirements.txt
+
+# 
+RUN pip install --no-cache-dir --upgrade -r /code/requirements.txt
+
+# 
+COPY ./app /code/app
+
+# 
+CMD ["uvicorn", "app.app:app", "--host", "0.0.0.0", "--port", "80"]
```

Comment:
is this a good idea?

Reply:
#roy

Reply:
passing the CMD with the host and ports here, in the dockerfile (There's also a docker-compose file)